docs/FAQ.md

```diff
@@ -35,3 +46,4 @@ If your question wasn't answered, feel free to ask on [Arduino CLI's forum board
 [screen]: https://www.gnu.org/software/screen/manual/screen.html
 [putty]: https://www.chiark.greenend.org.uk/~sgtatham/putty/
 [monitor command]: commands/arduino-cli_monitor.md
+[configuration parameters]: https://arduino.github.io/arduino-cli/dev/pluggable-monitor-specification/#describe-command
```

Comment:
```suggestion
[configuration parameters]: pluggable-monitor-specification.md#describe-command
```

Use relative link so that the target of the link will be of the same instance of the documentation as the page that was the source of the link.

Link to source file so that it will be functional when viewing the Markdown file directly in addition to when using the documentation website (MkDocs automatically converts the link to point to the appropriate HTML page in the generated website).